e2e/src/utils/utils.js

```diff
@@ -195,9 +195,7 @@ export async function quickOnboarding(mnemonic = SAMPLE_BACKUP_KEY) {
 
     // Assert on Wallet Home Screen
     await expect(element(by.id('SendOrRequestBar'))).toBeVisible()
-  } catch (e) {
-    console.error(e)
-  }
+  } catch {} // Don't throw an error just silently continue
```

Comment:
Why? 😄

Reply:
Keeps the Detox logs from having errors that aren't really errors. In some places we try quickOnboarding, but quickly arrive at this catch block if emulator's previous test already completed onboarding.